File: src/main/java/com/hardnets/coop/model/dto/request/userSubsidyRequest/Decree.java
package com.hardnets.coop.model.dto.request.userSubsidyRequest;

import lombok.Getter;
import lombok.Setter;

import java.util.Date;

import jakarta.validation.constraints.NotNull;

@Getter
@Setter
public class Decree {
    @NotNull
    private Integer number;

    @NotNull
    private Date publication;
}

File: src/main/java/com/hardnets/coop/model/dto/municipal/Decree.kt
package com.hardnets.coop.model.dto.municipal

import java.util.*

/**
 * Decreto que apruba el pago del subsidiado
 * @param number Número de decreto municipal que aprueba el subsidio => Código: NUM-DEC
 * @param approvalDate Fecha de aprobación del decreto => Código: FEC-DEC
 */
data class Decree(val number: Long?, val approvalDate: Date?)